### ISSUE TITLE
Some files missing from SMB folder display

### ISSUE THREAD

**Issue by sbuller** [files-community/Files#12626]
### Description

A few files have disappeared from a folder on a network share. They're still present when viewed in Explorer, the search field can still find them, and they reappear when the folder is copied to the desktop. I can't find any difference of permission or other metadata wrt. the other files. I've enabled showing hidden files and disabled thumbnails. Files still doesn't show them.
![files screenshot](https://github.com/files-community/Files/assets/8889/c4f0dbd6-b146-4402-8f45-bdcfd7e973fd)

### Steps To Reproduce

No idea.

### Requirements

All files should be shown

### Files Version

2.5.0

### Windows Version

Windows 10 (21H2): 19044

### Log File

[debug.log](https://github.com/files-community/Files/files/11762212/debug.log)

**Comment by Josh65-2201** [comment]
Thanks for the report.

**Comment by hishitetsu** [comment]
Same thing happens to me. In my case, I often don't see the shortcut file. Sometimes they do show up if I reload or open the folder again.

**Comment by yaira2** [comment]
@hishitetsu is this still an issue in the latest preview?

**Comment by hishitetsu** [comment]
> @hishitetsu is this still an issue in the latest preview?

Yes, but I've found that shortcuts sometimes don't show up even on local disk. Perhaps it happens when the shortcut destination takes a long time to load.

**Comment by Josh65-2201** [comment]
@sbuller Is this still an issue on v4.0.11 https://files.community/download

**Comment by sbuller** [comment]
It's been a couple years since we've used Files here, but I can't seem to reproduce the problem. I'll close the issue.

**Comment by Josh65-2201** [comment]
Thanks for checking
